server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionPasswordCache.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.connection;
+
+import java.util.concurrent.TimeUnit;
+
+import org.springframework.stereotype.Component;
+
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.Caffeine;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+/**
+ * @author jingtian
+ * @date 2024/3/27
+ */
+@Component
+public class ConnectionPasswordCache {
+
+    private final Cache<DatabaseUserIdentity, String> databaseUserIdentity2Password =
+            Caffeine.newBuilder().maximumSize(1000).expireAfterWrite(10, TimeUnit.HOURS).build();
```

Comment:
maxiumSize 10000;
expireAfterRead;

Reply:
Use expireAfterWrite(10, TimeUnit.HOURS), this expiration policy is used because DatabaseSyncManager accesses the cache every 3 minutes, if use expireAfterRead the cache will never expire.